src/main/java/com/suse/saltstack/netapi/event/EventStream.java

```diff
@@ -64,13 +72,11 @@ public EventStream() {
      *
      * @param config Contains the necessary details such as EndPoint URL and
      * authentication token required to create the WebSocket.
+     * @throws SaltStackException in case of an error during stream initialization
      */
-    public EventStream(ClientConfig config) {
-        try {
-            initializeStream(config);
-        } catch (SaltStackException e) {
-            e.printStackTrace();
-        }
+    public EventStream(ClientConfig config) throws SaltStackException {
+        maxMessageLength = config.get(ClientConfig.WEBSOCKET_MAX_MESSAGE_LENGTH);
```

Comment:
@isbm: No, @AdamMajer's proposed fix actually allows to remove the `maxMessageLength > 0` check, while keeping the feature of disabling the limit by setting 0 or -1. Setting the ultimate message length limit to `Integer.MAX_VALUE` makes sense to me, implemented with d62f995546a256739e1b89afdfa4f6bd3a145136.